libbpf-tools/execsnoop.c

```diff
@@ -304,6 +307,13 @@ int main(int argc, char **argv)
 	obj->rodata->max_args = env.max_args;
 	obj->rodata->filter_cg = env.cg;
 
+	buf = bpf_buffer__new(obj->maps.events);
```

Comment:
Have you tested it locally ?

Reply:
Yes, but poorly it turned out. Let me push my latest changes.

Reply:
Fixed this one.